src/atoms/cc-input-text.js

```diff
@@ -359,12 +386,30 @@ export class CcInputText extends LitElement {
           padding-bottom: 0.35em;
         }
 
+        :host([inline]) label {
+          flex-direction: column;
+          gap: 0;
+          grid-area: label;
+          justify-content: center;
+          line-height: normal;
+          padding: 0;
+        }
+        
+        :host([inline][multi]) label {
+          /* Allows the label text to be aligned with the first line of the input. */
+          height: 2em;
```

Comment:
this is not great but it's the only way I found to do this:
![image](https://user-images.githubusercontent.com/100240294/176459212-58111e4b-1c01-42ac-960e-fa857f4784a1.png)
![image](https://user-images.githubusercontent.com/100240294/176459274-4ac72d6d-5b42-49e4-86d3-be1c874c0cf4.png)